packages/prettier/CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# 2.1.1
+
+- (bug) Allow use in VSCode
+- (improvement) Improve documentation
+- (internal) Write prettier config as json
```

Comment:
we need to adapt the changelog to the format that changeset used (see changelog in other packages of this lib)